R/cellrangerCall.R

```diff
@@ -0,0 +1,23 @@
+cellrangerCall <- function(m, exp.cells = 3000, upper = 0.99, lower = 0.1)
+# A function to call cells on library size, as performed by CellRanger
+# 
+# written by Jonathan Griffiths
+# created 4 January 2018
+{
+  
+  if(upper > 1 | upper < 0){
+    stop("'upper' should be a numeric value between 0 and 1")
+  }
+  
+  if(lower > 1 | lower < 0){
+    stop("'lower' should be a numeric value between 0 and 1")
+  }
+  
+  libs <- Matrix::colSums(m)
+  
+  top <- libs[order(libs, decreasing = TRUE)][1:exp.cells]
```

Comment:
1. Never use `1:X`, always use `seq_len(X)`.
2. Slight increase in efficiency by doing `libs[order()[seq_len(exp.cells)]]`.